recipes/llvmdev/meta.yaml

```diff
@@ -60,3 +60,5 @@ extra:
     - inducer
     - pitrou
     - jakirkham
+    - tkelman
```

Comment:
It would be good to understand what those requirements are. Though maybe that discussion could happen in a new issue instead of an outdated diff.